src/app/(auth)/signin/page.tsx

```diff
@@ -0,0 +1,117 @@
+"use client";
+import { useFormik } from "formik";
+import { string } from "yup";
+import AuthForm from "@/components/organism/auth-form";
+import { login } from "@/models/services/kanvas/mutations/auth";
+import { useRouter } from "next/navigation";
+
+const emailSchema = string().required().email();
```

Comment:
Suggestion: for a cleaner way to do the implementation of the formik and yup validation we can use this approach 

```typescript
import * as yup from 'yup';
  const initialValues = {
    email: '',
    password: '',
  };
  
    const validationSchema = yup.object().shape({
    email: yup.string().email().required(),
    password: yup.string().required(),
  });


  async function onSubmit(values: typeof initialValues) {
    try {
      ....
    } catch {
      ....
    }
  }
  const {
    values,
    handleChange,
    handleSubmit,
    errors,
    setErrors,
    isSubmitting,
  } = useFormik({
    initialValues,
    validateOnBlur: false,
    validateOnChange: false,
    validationSchema,
    onSubmit,
  });
```